lib/smartSurveyAPICall.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const axios = require('axios');
+const querystring = require('querystring');
+const SmartSurveyClient = require('smartsurvey-client')
+
+module.exports = {
+  getResponses: (baseUrl, surveyID, endPoint, token, tokenSecret) => {
+    querystring.stringify({ foo: 'bar', baz: ['qux', 'quux'], corge: '' });
+    const apiUrlPart = querystring.stringify({'api_token':token, 'api_token_secret': tokenSecret});
+    axios.get(`${baseUrl}/${surveyID}/${endPoint}?${apiUrlPart}`)
+      .then(function (response) {
+        console.log(response);
+      })
+      .catch(function (error) {
+        console.log(error);
+      });
```

Comment:
in this particular scenario you could actually do `.catch(console.log)` or `console.error`